### DISCUSSION TITLE
IT modules with more than 99 samples. Is this OK?

### DISCUSSION THREAD

**Discussion by polyzium** [schismtracker/schismtracker#445] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22579935/35589649-97e8ca14-0615-11e8-9a15-898be497db3d.png)
![image](https://user-images.githubusercontent.com/22579935/35589695-bdfc356a-0615-11e8-8331-af3abc89f833.png)

**Comment by jangler** [comment, marked as answer]
It's an OpenMPT extension that Schism supports in some capacity. Does that answer your question?

**Comment by polyzium** [comment]
Oh, sorry, didn't know that. My bad. I thought it's weird because it displays 100+ samples as H0+ and so on. Displaying it as 100+ would make more sense, obviously.

**Comment by jangler** [comment]
Yeah, I'm also not sure what the logic behind the numbering scheme is, especially since there *is* room for a third digit. But I'm not really concerned about it either.

**Comment by sagamusix** [comment]
> especially since there is room for a third digit.

But not in the pattern editor, and this numbering scheme keeps the pattern and sample editor consistent. Whether a different scheme starting from A rather than H would have made more sense is another question, but interestingly, entering A in the instrument column in the pattern editor automatically turns it into H, B turns into I, etc...
